Packs/ContentTesting/Scripts/UnitTestCase/UnitTestCase.py

```diff
@@ -1,14 +1,15 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
+# Final Test: 6.10
 import io
 
 
 def main():
-    args = demisto.args()
-    testName = args.get("testName", "")
-    listName = args.get("listName", "")
-    addAfter = args.get("addAfter", "")
     try:
+        args = demisto.args()
```

Comment:
Why did you remove these 4 lines in to the 'try' block?
It does not need to be there beucus there is no scenario for it to fail, and the way it is now will not execute,  because in the bottom the 'except Exception' is out of the 'try' block  and we call 'testName', which is inside of the 'try' block.

Reply:
Not following you? I did not remove 4 lines and the Exception is still valid since call DemistoException.

Reply:
My apologies, there was a bug in the process of uploading your contribution. This caused some files to be changed that you did not change. That will be fixed.